src/assignment/views/AssignmentCreate.tsx

```diff
@@ -245,12 +248,35 @@ const AssignmentCreate: FunctionComponent<DefaultSecureRouteProps> = ({
 						}}
 					/>
 				),
+				// Only assignments get to pick colors
+				actions: (item, i) =>
+					item && (
+						<>
+							<ListSorterColor item={item} />
+							<ListSorterPosition item={item} i={i} />
+							<ListSorterSlice item={item} />
+						</>
+					),
 			},
 			listSorterItem: {
 				onSlice: (item) => {
 					confirmSliceModal.setEntity(item);
 					confirmSliceModal.setOpen(true);
```

Comment:
OPTIONAL: setEntity and setOpen could get out of sync
you could also opt for only setEntity and infer the isOpen if the Entity is set: const isOpen = !!entity

Reply:
Won't change this because it's not touched by BaO